app/main/views/reset_password.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+
+import six
+
+from flask import abort, current_app, flash, redirect, render_template, url_for, Markup
+
+from dmutils.user import User
+from dmutils.email import generate_token, decode_password_reset_token, send_email
+from dmutils.email.exceptions import EmailError
+
+from .. import main
+from ..forms.auth_forms import EmailAddressForm, ChangePasswordForm
```

Comment:
Not for this PR, but there is now an `EmailAddressForm` in utils that we should move to using for all our email address form fields.